mLRS/Common/leds.h

```diff
@@ -39,7 +39,9 @@ class tLEDs
 #ifndef DEVICE_HAS_SINGLE_LED_RGB
             if (!blink) { led_green_toggle(); led_red_toggle(); }
 #endif
+#ifdef DEVICE_HAS_SINGLE_LED_RGB
```

Comment:
I'm not sure yet if I should like the introduction of yet another color, but could you in Tick_ms() separate the code into the three sections
#ifdef DEVICE_HAS_SINGLE_LED
....
#elif defined DEVICE_HAS_SINGLE_LED_RGB
....
#esle
...
#endif 
it's getting complicated to follow the flow of the code

my hesitation with yet another blink pattern is that I don't like having that many different ones ... noone can memorize them all ... example, could you tell all ELRS rgb blink patterns without having to look em up?
is there a specific reason to not go with red-green blink?

Reply:
More of a fun one, no plans for any more blink patterns :).   

At least in English the alliteration makes sense to me, **_b_** lue for _**b**_ ind